sci-electronics/kicad/kicad-9.0.0-r1.ebuild

```diff
@@ -42,6 +60,7 @@ COMMON_DEPEND="
 	dev-db/unixODBC
 	dev-libs/boost:=[context,nls]
 	dev-libs/libgit2:=
+	dev-libs/protobuf[protobuf]:=
```

Comment:
This is invalid syntax. `pkgcheck scan` should fail on it.

Reply:
Yes, I noticed immediately after push, fixed already.